chart/templates/NOTES.txt

```diff
@@ -6,4 +6,8 @@ Check out our docs at https://rancher.com/docs/rancher/v2.x/en/
 
 Browse to https://{{ .Values.hostname }}
 
+If this is the first time you installed Rancher, run this command to get link to setup initial password:
+
+$ echo https://{{ .Values.hostname }}/dashboard/?setup=$(kubectl get secret --namespace {{ .Release.Namespace }} bootstrap-secret -o jsonpath="{.data.bootstrapPassword}" | base64 --decode)
```

Comment:
Can we print out the actual password from templating so this can actually be a directly clickable link?  This is your first interaction with the product, so the less friction the better.

Reply:
Tried many times... but helm doesn't seem to have a good way to define a random number elsewhere and import it to note.txt, also that will get re-generated everytime everytime chart is installed/upgraded.